plugins/modules/elb_application_lb.py

```diff
@@ -54,8 +54,33 @@
   http2:
     description:
       - Indicates whether to enable HTTP2 routing.
-      - Defaults to C(false).
+      - Defaults to C(true)
+    type: bool
+  http_desync_mitigation_mode:
+    description:
+      - Determines how the load balancer handles requests that might pose a security risk to an application.
+      - Defaults to C('defensive')
+    type: str
+    choices: ['monitor', 'defensive', 'strictest']
+    version_added: 4.0.0
```

Comment:
the question is, if we introduce this in 3.2.0 and make community.aws depends on amazon.aws 3.2.0 because of https://github.com/ansible-collections/amazon.aws/pull/696

Reply:
@alinabuzachis will be amazon.aws 3.2.0 released before community.aws 3.2.0?

Reply:
We can release it even before. There's no problem from my side, I can handle it. And @mandar242 can release the 2.3 for amazon.aws. What do you think @jillr?